src/sentry/conf/server.py

```diff
@@ -1108,7 +1108,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     # Enable rate limits for inviting members.
     "organizations:invite-members-rate-limits": True,
     # Enable new issue actions on issue details
-    "organizations:issue-actions-v2": False,
+    "organizations:issue-actions-v2": True,
```

Comment:
can we just remove the flag?

Or is that what you're starting to do here

Reply:
Making the flag true by default is a good first step to removing the flag.

Reply:
yeah marking true so ST and stuff gets it. looks like there's some acceptance tests failing